gauges/tables.go

```diff
@@ -260,3 +260,40 @@ func (g *Gauges) TableScans() *prometheus.GaugeVec {
 
 	return gauge
 }
+
+var hotUpdatesQuery = `
+SELECT relname
+	 , n_tup_hot_upd 
+  FROM pg_stat_user_tables
+`
+
+type tableHotUpdates struct {
+	Name       string  `db:"relname"`
+	HotUpdates float64 `db:"n_tup_hot_upd"`
+}
+
+func (g *Gauges) HOTUpdates() *prometheus.GaugeVec {
+	var gauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name:        "postgresql_hot_updates",
+			Help:        "Heap-Only tuple updates metric",
```

Comment:
here we can use a better description of what this metric is, something like `Number of rows Heap-Only tuple updated` (get from PostgreSQL documentation)